frontend/src/components/NewShoot/GNewShootInfrastructureDetails.vue

```diff
@@ -272,91 +276,55 @@ export default {
     }
   },
   validations () {
-    return this.validators
+    return {
+      region: withFieldName('Region', {
+        required,
+      }),
+      networkingType: withFieldName('Networking Type', {
+        required: requiredIf(!this.workerless),
+      }),
+      loadBalancerProviderName: withFieldName('Cluster Name', {
+        required: requiredIf(function () {
+          return this.infrastructureKind === 'openstack'
```

Comment:
We use function not arrow function. Is this still needed? 
Can't we have local helper functions:
```js
const requiresInfrastructure = infrastructureKind => {
  return requiredIf(() => this.infrastructureKind === infrastructureKind)
}
// usage
const rules = {
        required: requiresInfrastructure('openstack')
}
```
This will also removes one level of nesting